backend/app/models.py

```diff
@@ -2,5 +2,39 @@
 
 from . import db
 from werkzeug.security import generate_password_hash, check_password_hash
-
+from sqlalchemy.dialects.postgresql import UUID
+import uuid
+import datetime
 # create model classes here
+
+class Donation(db.Model):
+    __tablename__='donations'
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    name = db.Column(db.String(64), nullable=False)
+    email = db.Column(db.String(64), unique=True, nullable=False)
+    date = db.Column(db.DateTime(), default=datetime.utcnow)
+    donationSource = db.Column(db.String(128), nullable=False)
+    event = db.Column(db.String(128), nullable=False)
```

Comment:
this was probably my bad but I don't think there will always be an event associated with a donation - so take out `nullable=False`